config/mcus/can_octopus.cfg

```diff
@@ -0,0 +1,58 @@
+[mcu]
+##--------------------------------------------------------------------
+# This board work by using a serial connection by default.
+# If you want to use CAN, invert the commented lines here
+
+# serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+# canbus_uuid: change-me-to-the-correct-canbus-id
+##--------------------------------------------------------------------
+
+
+[board_pins mcu]
+mcu: mcu
+aliases:
+
+# steppers
+	X_STEP=PF13, X_DIR=PF12, X_ENABLE=PF14, X_TMCUART=PC4,
+	Y_STEP=PG0, Y_DIR=PG1, Y_ENABLE=PF15, Y_TMCUART=PD11, Y_STOP=PG9,
+	E_ENABLE=PD4,
+	Z_STEP=PF11, Z_DIR=PG3, Z_ENABLE=PG5, Z_TMCUART=PC6, Z_STOP=PG10,
+	Z1_STEP=PG4, Z1_DIR=PC1, Z1_ENABLE=PA0, Z1_TMCUART=PC7,
+	Z2_STEP=PF9, Z2_DIR=PF10, Z2_ENABLE=PG2, Z2_TMCUART=PF2,
+	Z3_STEP=PC13, Z3_DIR=PF0, Z3_ENABLE=PF1, Z3_TMCUART=PE4,
+
+# bed
+	BED_HEATER=PA1, BED_TEMPERATURE=PF3, 
+
+# sensor
+	CHAMBER_TEMPERATURE=PF6, ELECTRICAL_CABINET_TEMPERATURE=PF5,
+
+# recirculating filter
+	FILTER_FAN=P2.7,	
+	
+# Controller Fans
+	CONTROLLER_FAN1=PA8, CONTROLLER_FAN2=PE5,
+
+# EXHAUST FAN
+    EXHAUST_FAN=PD12,
+
+#lights
+    LIGHT_OUTPUT=PD15,
+    LIGHT_NEOPIXEL=PB6,
+
+# runout_sensor
+    RUNOUT_SENSOR=PG11,
+
+# EXP1 header
+    EXP1_1=PE8, EXP1_2=PE7,
+    EXP1_3=PE9, EXP1_4=PE10,
+    EXP1_5=PE12, EXP1_6=PE13,    # Slot in the socket on this side
+    EXP1_7=PE14, EXP1_8=PE15,
+    EXP1_9=<GND>, EXP1_10=<5V>,
+
+# EXP2 header
+    EXP2_1=PA6, EXP2_2=PA5,
+    EXP2_3=PB1, EXP2_4=PA4,
+    EXP2_5=PB2, EXP2_6=PA7,      # Slot in the socket on this side
+    EXP2_7=PC15, EXP2_8=<RST>,
+    EXP2_9=<GND>, EXP2_10=<5V>
```

Comment:
This file is not needed as someone could use the normal `octopus.cfg` file. So I think you can remove it :)

Reply:
Yes you are right, but since a lot of pins would move to the CANbus, won't it be confusing to the user?

Especially when changing pins to "toolhead:". Pins which are refered to the standard octopus won't pop up as fault and the user will have to search through different cfg files.

Reply:
Okay I have tought about it. Yes totally doable.
But you would need a documentation of every toolhead pinout documented.
Also you will need to make sure that the user understands which pinout are now free on the "mcu" board.

As you can see, because of the canbus. My fan & tempreature pins are different.

Just tell me which way you prefair and I will work on the documentation/comments.

It would also be possible to just comment which PINS should be commented/replaced when using Canbus.

Essentially combining both cfgs